source/includes/_account.md

```diff
@@ -2117,7 +2117,7 @@ If successful, returns a 201 status code. In addition, a `Location` header is re
 
 ## Payment Method
 
-These endpoints allow you to manage the payment methods for an account. See section [Payment Method](#payment-method) for details on payment methods.
+These endpoints allow you to manage the payment methods for an account. See section [Payment Method](https://github.com/killbill/slate/payment-method.html) for details on payment methods.
```

Comment:
I think you can do a relative link, so that it also works locally when running `bundle exec middleman server`: try `payment-method.html` instead